src/Components/Publishing/Article.tsx

```diff
@@ -26,7 +26,7 @@ export interface ArticleProps {
     panel: object
     canvas: any
   }
-  showTooltips?: boolean
+  showToolTips?: boolean
```

Comment:
tooltip is actually a word btw ;) sounds good though

Reply:
Yeah I know it is a word, but wanted to just have one way of doing it -- and this one makes the linter happier.